esx_service/utils/vmdk_utils.py

```diff
@@ -358,7 +362,7 @@ def check_volumes_mounted(vm_list):
         vm = vmdk_ops.findVmByUuid(vm_id)
         if vm:
             for d in vm.config.hardware.device:
-                if check_docker_volume(d):
+                if find_dvs_volume(d):
```

Comment:
Since return from find_dvs_volume() is unused does it need to return any data at all?